### PR TITLE
add(translation)component/breadcrumb

### PR DESCRIPTION
**Pull request by forthedamn**
### translation breadcrumb
- 1. add index.en_US.md
- 2. rename index.md to index.zh_CN.md
- 3. add the translation in demo file

### CODE REVIEW

**Comment by benjycui** [comment]
@waywardmonkeys could you help us to review this translation?

File: components/breadcrumb/demo/separator.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+  en-US: Separator
 ---
 
+## zh-CN
+
 使用 `separator=">"` 可以自定义分隔符。
 
+## en-US
+
+Use `separator=">"` to custom separator.
```
Comment (waywardmonkeys) [inline components/breadcrumb/demo/separator.md]: The separator can be customized by setting the `separator` property: `separator=">"`

File: components/breadcrumb/demo/withIcon.md [hunk 2]
```diff
@@ -1,10 +1,18 @@
 ---
 order: 1
-title: 带有图标的
+title:
+  zh-CN: 带有图标的
+  en-US: With Icon
```
Comment (waywardmonkeys) [inline components/breadcrumb/demo/withIcon.md]: With an Icon

File: components/breadcrumb/demo/withIcon.md [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
 ---
 
+## zh-CN
+
 图标放在文字前面。
 
+## en-US
+
+Icon should in the front of text.
```
Comment (waywardmonkeys) [inline components/breadcrumb/demo/withIcon.md]: The icon should be placed in front of the text.

File: components/breadcrumb/demo/basic.md [hunk 4]
```diff
@@ -1,10 +1,18 @@
...
+## zh-CN
+
 最简单的用法。
 
+## en-US
+
+The simplest use
+
 ````jsx
 import { Breadcrumb } from 'antd';
```
Comment (waywardmonkeys) [inline components/breadcrumb/demo/basic.md]: I suspect you should translate the Chinese text in the example blocks of code in here as well.

File: components/breadcrumb/demo/basic.md [hunk 5]
```diff
@@ -1,10 +1,18 @@
 ---
 order: 0
-title: 基本
+title:   
+  zh-CN: 基本
+  en-US: Basic
```
Comment (waywardmonkeys) [inline components/breadcrumb/demo/basic.md]: Basic Usage

File: components/breadcrumb/demo/router.md [hunk 6]
```diff
@@ -1,11 +1,19 @@
 ---
 order: 2
 iframe: true
-title: 路由
+title: 
+  zh-CN: 路由
+  en-US: Router
```
Comment (waywardmonkeys) [inline components/breadcrumb/demo/router.md]: React Router Integration

File: components/breadcrumb/demo/separator.md [hunk 7]
```diff
@@ -1,10 +1,18 @@
 ---
 order: 3
-title: 分隔符
+title: 
+  zh-CN: 分隔符
+  en-US: Separator
```
Comment (waywardmonkeys) [inline components/breadcrumb/demo/separator.md]: Configuring the Separator

File: components/breadcrumb/index.en-US.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+---
+category: Components
+type: Navigation
+title: Breadcrumb
+---
+
+Breadcrumb displays the current page in the system hierarchy position, and can back to previous.
```
Comment (waywardmonkeys) [inline components/breadcrumb/index.en-US.md]: A breadcrumb displays the current location within a hierarchy. It allows going back to states higher up within the hierarchy.

File: components/breadcrumb/index.en-US.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+## When to use
+
+- When the system has more than two hierarchy,
+- When you need to inform the user, "where are you",
+- When you need to back to previous.
```
Comment (waywardmonkeys) [inline components/breadcrumb/index.en-US.md]: - When the system has more than two layers in a hierarchy.
- When you need to inform the user of where they are.
- When the user may need to navigate back to a higher level within the hierarchy.

Comment (waywardmonkeys) [inline components/breadcrumb/index.en-US.md]: Using the word hierarchy here sounds wrong to me, but I'm not sure of a good alternative at the moment. I'm not quite awake yet.

Ideas, @afc163 ?

**Comment by waywardmonkeys** [comment]
@benjycui Sure thing and done for a first pass. (Feel free to ping me like this on bugs where my input is useful!)

File: components/breadcrumb/demo/basic.md (hunk 4 above)
Comment (forthedamn) [inline components/breadcrumb/demo/basic.md]: OK,thanks for your advice.

File: components/breadcrumb/index.en-US.md (hunk 9 above)
Comment (afc163) [inline components/breadcrumb/index.en-US.md]: How about `When your web application has multi-layer architecture`?

File: components/breadcrumb/demo/basic.md (hunk 4 above)
Comment (forthedamn) [inline components/breadcrumb/demo/basic.md]: @afc163 Could I translate like this :

## zh-CN

描述

```jsx
Chinese text in the example block
```

## en-US

detail

```jsx
English text in the example block
```

**Comment by forthedamn** [comment]
@waywardmonkeys New commit has been pushed, could you help me to review the translation?

**Comment by benjycui** [comment]
It's good enough. 

If anyone want to revise this documentation, just open a new pull request, THX!

**Pull request merged**